provider/online_test.go

```diff
@@ -39,24 +39,34 @@ func TestOnlineStores(t *testing.T) {
 		"TypeCasting":        testTypeCasting,
 	}
 
+	//Redis (Mock)
 	miniRedis := mockRedis()
 	defer miniRedis.Close()
 	mockRedisAddr := miniRedis.Addr()
 	redisMockConfig := &RedisConfig{
 		Addr: mockRedisAddr,
 	}
+
+	//Redis (Live)
 	redisPort := os.Getenv("REDIS_PORT")
 	liveAddr := fmt.Sprintf("%s:%s", "localhost", redisPort)
 	redisLiveConfig := &RedisConfig{
 		Addr: liveAddr,
 	}
 
+	//Cassandra
 	cassandraAddr := "localhost:9042"
 	cassandraConfig := &CassandraConfig{
 		Addr:        cassandraAddr,
 		Consistency: gocql.One,
 	}
 
+	//Firestore
+	projectID := "snappy-density-352219"
```

Comment:
Need to import the API Key too right? Well talk later about how to add the api key to testing